diff --git a/node_modules/react-native-html-to-text/lib/HtmlText.js b/node_modules/react-native-html-to-text/lib/HtmlText.js
index 00dec90..980d2f7 100644
--- a/node_modules/react-native-html-to-text/lib/HtmlText.js
+++ b/node_modules/react-native-html-to-text/lib/HtmlText.js
@@ -6,6 +6,7 @@ import {
   Text,
   View,
   Linking,
+  Appearance
 } from 'react-native';
 
 import React, {Component} from 'react';
@@ -63,7 +64,8 @@ function processNode(node, parentKey, props) {
 
   if (isText(nodeName)) {
     const key = `${parentKey}_text`
-    return (<Text key={key} style={props.style}>{node.value}</Text>)
+    const scheme = Appearance.getColorScheme()
+    return (<Text key={key} style={{color: scheme == "light" ? "#000" : "#fff"}}>{node.value}</Text>)
   }
 
   if (isInlineElement(nodeName)) {
diff --git a/node_modules/react-native-html-to-text/lib/constants.js b/node_modules/react-native-html-to-text/lib/constants.js
index 30ef905..b2f6850 100644
--- a/node_modules/react-native-html-to-text/lib/constants.js
+++ b/node_modules/react-native-html-to-text/lib/constants.js
@@ -1,15 +1,17 @@
 
 import {
   StyleSheet,
+  Appearance
 } from 'react-native';
 
+const scheme = Appearance.getColorScheme()
 export const BLOCK_ELEMENTS = ["blockquote", "div", "h1", "h2", "h3", "h4", "h5", "h6", "hr", "ol", "p", "pre", "ul", "li"]
 export const INLINE_ELEMENTS = ["b", "i", "em", "strong", "a", "br", "q", "span", "sub", "sup", "u"]
 export const DEFAULT_STYLES = StyleSheet.create({
   a: {
-    textDecorationLine: 'underline',
+    textDecorationLine: 'none',
     textDecorationStyle: 'solid',
-    color: "blue",
+    color: "#3e84ed",
   },
   b: {
     fontWeight: 'bold'
@@ -57,7 +59,7 @@ export const DEFAULT_STYLES = StyleSheet.create({
     fontStyle: 'italic'
   },
   p: {
-    marginBottom: 12,
+    marginBottom: 12
   },
   pre: {
 
